### ISSUE TITLE
Rotation support

### ISSUE THREAD

**Issue by bugsend** [i-schuetz/SwiftCharts#144]
Thank you for tremendous work you did and for sharing results with the public! Really appreciate!

Great work!

I've just mentioned that Charts ( at least those I've been experimenting with ) do not support device' rotation. I mean they do not scale appropriately.

Is this a feature or a bug ? And how to implement it manually in case this is a design decision you've made ? ( call setNeedsDisplay on _charts_ var inside viewWillTransitionToSize doesn't work ).

Thank you !

**Comment by i-schuetz** [comment]
Thanks :)  https://github.com/i-schuetz/SwiftCharts/issues/105

**Comment by bugsend** [comment]
Thanks for spotting solution. If a got the point right, main idea is to detect orientation change event and then call "rotate()" method. 

But

In my point I use Charts as a View in UITableViewCell and such approach bring to result at all. Maybe you have some knowledge on how to force table's cell redraw ( calling chart.view.setNeedsDisplay() doesn't help either ).

Thank you in advance !

**Comment by bugsend** [comment]
Well, it seems like I've found the solution. 
It looks smth like this:

1. `self.quickChartsTableView.beginUpdates()`
2. `self.quickChartsTableView.reloadRowsAtIndexPaths([NSIndexPath(forRow: 0, inSection: 0), NSIndexPath(forRow: 1, inSection: 0), NSIndexPath(forRow: 2, inSection: 0)], withRowAnimation: .Automatic)`
3. `self.quickChartsTableView.endUpdates()`

Works in my case since I have defined number of charts in a table.
